run_healthcheck.py

```diff
@@ -78,9 +84,8 @@ def sendMessage(channel, message,numMessages = 1):
 verify = sendMessage(dm," emergency {}".format(auth['user']))
 if verify.find("verify") == -1:
     raise RuntimeError,"Unexpected response to emergency: {}".format(verify)
-# Bot sends 3 messages in response, we need all of them and want the last
-# message
-info = sendMessage(dm," YES ",numMessages=3)[-1]['text']
+# Bot sends 3 messages in response, we need all of them and want the first message
+info = sendMessage(dm," YES ",numMessages=3, wait=5)[0]['text']
 logger.info(info)
 if info.find(dt) == -1:
```

Comment:
@callen314 maybe the thing here is to make this check more intelligent: loop over everything returned and look for dt. It's a little stringent especially if the order is not guaranteed.

Reply:
Something like:

```python
found = [x for x in in sendMessage(dm,....,numMessages=3) if x['text'].find(dt) != -1]
if not found:
   ....
```